styles/Home.module.scss

```diff
@@ -1,27 +1,19 @@
 @import "media";
 
-.container {
-  width: 100%;
-
-  .cards__wrapper {
-    display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(350px, 1fr));
-    grid-auto-rows: 1fr;
-    gap: 30px 45px;
-    justify-items: center;
-    padding: 0 25px;
-    margin: 0 auto;
-    max-width: 1769px;
-    margin-bottom: 100px;
-  }
+.cards__wrapper {
```

Comment:
Please separate the changes of the grid to other PR.

Reply:
https://github.com/Pull-Request-Community/pull-request-community-website/pull/218 
done!